compute/pbs/backend.go

```diff
@@ -27,3 +38,78 @@ func NewBackend(conf config.Config, reader tes.ReadOnlyServer, writer events.Wri
 func extractID(in string) string {
 	return in
 }
+
+type job struct {
+	JobID      string `xml:"Job_Id"`
+	JobState   string `xml:"job_state"`
+	ExitStatus int    `xml:"exit_status"`
+}
+
+type xmlRecord struct {
+	XMLName xml.Name `xml:"Data"`
+	Job     []job
+}
+
+func mapStates(ids []string) ([]*compute.HPCTaskState, error) {
+	var output []*compute.HPCTaskState
+
+	idSet := make(map[string]interface{})
+	for _, i := range ids {
+		idSet[i] = nil
+	}
+
+	cmd := exec.Command("qstat", "-x")
+	stdout, err := cmd.Output()
+	if err != nil {
+		return nil, err
```

Comment:
can you wrap these errors? I've noticed with other pieces of code with unwrapped errors that I struggle to hunt down the source line in funnel